Dockerfile.generateenv

```diff
@@ -0,0 +1,6 @@
+FROM python:3.9-slim-bullseye
+ENV PATH="/root/.local/bin:$PATH"
+RUN pip install jinja2>=3.0.3
+WORKDIR /opt/todoapp
+COPY scripts/generate_env.py .
+ENTRYPOINT [ "python", "generate_env.py" ]
```

Comment:
I can't say I really understand the purpose of this stage; looks like an interesting challenge but it feels like this is a harder solution to creating a .env file than just pasting the appropriate values into a copy of the `.env.template` file and copying them into the appropriate folder!

Reply:
Yes - I agree it's a bit overkill, just wanted to try an example of a one-off command that changes the host filesystem